File: Python Language/Q23.py
# Program to calculate Area and Circumference of a circle

import math

radius = float(input("Enter the radius of the circle: "))

# Formulas
# Area = pi * r^2
area = math.pi * (radius ** 2)
# Circumference = 2 * pi * r
circumference = 2 * math.pi * radius

print(f"Area: {area:.2f}")
print(f"Circumference: {circumference:.2f}")

File: C++ Language/Q23.cpp
#include <iostream>
using namespace std;

int main() {
    float radius, area, circumference;
    const float PI = 3.14159; // Defining PI as a constant

    cout << "Enter the radius of the circle: ";
    cin >> radius;

    // Formulas
    area = PI * radius * radius;       // Area = pi * r^2
    circumference = 2 * PI * radius;   // Circumference = 2 * pi * r

    cout << "Area: " << area << endl;
    cout << "Circumference: " << circumference << endl;

    return 0;
}